dashboard/app/controllers/api/v1/pd/workshop_survey_report_controller.rb

```diff
@@ -1,20 +1,20 @@
-require 'controllers/api/v1/pd/workshop_score_summarizer'
+module Api::V1::Pd
+  class WorkshopSurveyReportController < ReportControllerBase
+    include WorkshopScoreSummarizer
+    load_and_authorize_resource :workshop, class: 'Pd::Workshop'
 
-class Api::V1::Pd::WorkshopSurveyReportController < Api::V1::Pd::ReportControllerBase
-  include WorkshopScoreSummarizer
-  load_and_authorize_resource :workshop, class: 'Pd::Workshop'
+    # GET /api/v1/pd/workshops/:id/aggregate_workshop_score
```

Comment:
Mind fixing this comment while you are here? It's workshop_survey_report not aggregate_workshop_score